token.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -euo pipefail
```

Comment:
```suggestion
```

There are lots of ways to fail, not all of which need to propagate up. Id rather be specific on which ones cause startup to fail since the container could throttle as a result. If theres a fail case that needs to cause this, make it specific with error messages etc